docs/dbt/index.rst

```diff
@@ -0,0 +1,65 @@
+Cosmos - dbt Support
+====================
+
+Cosmos allows you to render your dbt models as Airflow DAGs and Task Groups.
+
+.. toctree::
+    :maxdepth: 2
+    :caption: Sections
+
+    Installation Options <install-options>
+    Usage <usage>
+    Scheduling <scheduling>
+
+
+Quickstart
+----------
+
+Install the package using pip:
+
+.. code-block:: bash
+
+    pip install astronomer-cosmos[dbt.all]
+
+
+Create a DAG and import the :class:`cosmos.providers.dbt.DbtTaskGroup` class. The ``DbtTaskGroup`` operator requires a the name of your dbt project, an Airflow connection ID, a schema, and any additional arguments you'd like to pass to dbt.
+
+.. code-block:: python
+
+    from pendulum import datetime
```

Comment:
Will this repo parse the code whenever we make a change to the framework?

Reply:
Not in these .rst pages - there may be ways to link back to code examples but I don't think we need that for a first pass